docs/src/forc_client.md

```diff
@@ -1,3 +1,29 @@
 # forc-client
 
 Forc plugin for interacting with a Fuel node.
+
+## Signing transactions using `forc-wallet` CLI
+
+To submit the transactions created by `forc deploy` or `forc run`, you need to sign them first (unless you are using a client without utxo validation). To sign a transaction you can use `forc-wallet` CLI. This section is going to walk you through the whole signing process.
+
+To install `forc-wallet` please refer to `forc-wallet`'s [github repo](https://github.com/FuelLabs/forc-wallet).
```

Comment:
```suggestion
To install `forc-wallet` please refer to `forc-wallet`'s [github repo](https://github.com/FuelLabs/forc-wallet#forc-wallet).
```

perhaps link it to the README so that clicking on the link jumps to that section